index.js

```diff
@@ -39,11 +48,14 @@ function apiRequest(method, resource, payload, callback) {
       options.body = JSON.stringify(payload);
     }
     request(options, function(error, response, body) {
-      if (!error && response.statusCode === 200) {
-        callback(null, body ? JSON.parse(body) : {});
+      if (error || response.statusCode !== 200) {
+        callback(error ||
+          {
+            status: response.statusCode,
+            message: `${JSON.stringify(response, null, 2)}`
```

Comment:
Would prefer to have the actual object instead of the stringified version.  Why are we stringifying here?  Also make sure that classy-api-client and classy-pay-client are consistent (we get the same type of error style back) and consumable.

Reply:
Quick link: https://github.com/classy-org/classy-pay-client/blob/master/index.js